pkg/common/fflag/fflag.go

```diff
@@ -38,6 +38,10 @@ const (
 	// reattestable. In those cases the agent will still renew without reattesting.
 	FlagReattestToRenew Flag = "reattest_to_renew"
 
+	// FlagCAJournalInDatastore controls whether or not to store the CA journal
+	// of the server in the datastore.
+	FlagCAJournalInDatastore = "ca_journal_in_datastore"
```

Comment:
@MarcosDY are you suggesting that the flag should be true by default? That would be the process of deprecating a  feature flag, and we cannot do that in 1.8.x.
Feature flags are meant to be used by developers while features are still under development or cannot be turned on to all users (like this case, because the schema change required was introduced in 1.8.0).

@edwbuck There is no need to have this feature introduced as an experimental feature in 1.9.0. All users will have the schema changes needed when they either update to 1.9.0 or fresh install 1.9.0.
In 1.9.0, this will be the default behavior to all users.